benchbuild/project.py

```diff
@@ -17,21 +17,23 @@
 """
 import copy
 import logging
-from typing import Tuple, Optional, Mapping, Type
 import uuid
 from abc import abstractmethod
 from functools import partial
 from os import getenv
+from typing import Dict, List, Mapping, Optional, Tuple, Type
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No name 'Mapping' in module 'typing'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5028678)